content/docs/latest/howtorelease_27362106.md

```diff
@@ -412,17 +424,20 @@ Once [three PMC members have voted for a release](http://www.apache.org/foundati
 1. Tag the release and delete the release candidate tag. Do it from the release branch:
 
 ```
-git tag -s rel/release-X.Y.Z -m "HiveX.Y.Z release."
+git tag -s rel/release-X.Y.Z -m "Hive X.Y.Z release."
 git push origin rel/release-X.Y.Z
 git tag -d release-X.Y.Z-rcR
 git push origin :release-X.Y.Z-rcR
 ```
+**NOTE:**  
+If errors happen while "git tag -s", try to configure the git signing key by "git config user.signingkey your_gpg_key_id" then rerun the command.  
+This step (`git push origin rel/release-X.Y.Z`) will trigger the Hive Docker image build and upload to Docker Hub on the [Hive Action Page](https://github.com/apache/hive/actions/workflows/docker-GA-images.yml). If the image build fails, click **Re-run** on the Actions page to retry or manually build and upload it. Finally, verify whether the image has been successfully uploaded by checking the [Docker Hub](https://hub.docker.com/r/apache/hive/tags).
```

Comment:
This step is prone to failure. At the time, I had to click "Re-run" to retry before successfully building the Hive image. I'm not sure if manually building and uploading it complies with the rules. If it's non-compliant, the manually operated content should be deleted.   cc @dengzhhu653 @deniskuzZ